custom_news_feed/custom_news_feed.py

```diff
@@ -267,11 +277,27 @@ def display_news_content(self, news_json_file_path):
             QgsMessageLog.logMessage(u'Error Reading Config file, missing field ' + str(e),'Custom News Feed')
 
 
+    def toggle_message_hashfile(self, event):
+        """Toggles pinned messages resized/full-sized state"""
+        hash = hashlib.md5(str(self.current_pinned_message["Text"]).encode('utf-8')).hexdigest()
+        if self.check_hashfile(hash) == True :
```

Comment:
PP: Unnötigen Whitespace entfernen
```suggestion
        if self.check_hashfile(hash) == True:
```